app/dto/overnight.js

```diff
@@ -0,0 +1,23 @@
+const purgeSoftDeletedDto = (softDeletedResponse) => {
+  return {
```

Comment:
Is this DTO necessary? It seems to exactly replicate the structure of the original input data

Reply:
not entirely, the import data has a getter in it which we need to call and remove